Documentation/kubelet-wrapper.md

```diff
@@ -0,0 +1,39 @@
+# Kubelet Wrapper Script
+
+The kubelet has some unique requirements, so we need to be able to run the kubelet in an unconstrained environment. However, we also want to ship the kubelet as a container image to take advantage of all that has to offer (image discovery, signing/verification, management).
+
+The kubelet-wrapper is a helper-script shipped with CoreOS versions 960.0.0+. The script allows a deployer to easily run the kubelet as a container on the host system.
```

Comment:
CoreOS ships a wrapper script, `kubelet-wrapper`, which makes it very easy to run the kubelet under rkt. This script allows for future CoreOS releases to tweak the parameters of the kubelet, while allowing user-specified flags and the desired version of the kubelet to be passed to rkt. This script is currently shipping in CoreOS 960.0.0+ and will be included in all channels in the near future.